src/NUnitFramework/framework/Internal/Randomizer.cs

```diff
@@ -611,6 +611,25 @@ public decimal NextDecimal(decimal min, decimal max)
             return unchecked(raw % range + min);
         }
 
+        #endregion 
+
+        #region Guid
+
+        /// <summary>
+        /// Returns a random <see cref="Guid"/>.
```

Comment:
I am expecting people to want to know that it's some form of valid guid and which format. Absent information, it would seem entirely likely that someone had just done the naive thing and generated 16 random bytes and put them in a `Guid` and left it at that.

It seems like there is some value in acknowledging somehow that `Guid.NewGuid()` also generates type 4 guids because people will probably often be replacing that with this. But I suppose they could do their own research if they wanted to know.